### PR TITLE
update package.json of Fineuploader and add autoupdate config

### LINKED ISSUE #7707: [Request] Add updated Fineuploader v5.7.0
The latest version you have is 3.7.0:
https://cdnjs.com/libraries/file-uploader

Could you update it to the latest (currently 5.7.0)?
https://github.com/FineUploader/fine-uploader
http://fineuploader.com

Thank you

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7707 
@dannyxx001 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/FineUploader/fine-uploader
 * Official website (optional, not the repository): http://fineuploader.com/
 * NPM package url (optional): https://www.npmjs.com/package/fine-uploader
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 5861
   - Count of watchers: 309
   - Count of forks: 1414
 * NPM download stats (optional):
   - Downloads in the last day: 32
   - Downloads in the last week: 662
   - Downloads in the last month: 2960

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment

close #7707 , cc @rnicholus

### CODE REVIEW

**Comment by x09326** [comment]
@pvnr0082t 
Please combine to one commit.

**Comment by pvnr0082t** [comment]
@x09326 I have combined them. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t please clean up your commit message.

**Comment by pvnr0082t** [comment]
@x09326 I have cleaned up commit message. Please help me review again, thank you.

File: ajax/libs/file-uploader/package.json [hunk 1]
```diff
@@ -12,5 +12,20 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/valums/file-uploader.git"
```
Comment (PeterDaveHello) [inline ajax/libs/file-uploader/package.json]: Repo url should also been updated.

**Comment by PeterDaveHello** [comment]
Sorry that I didn't catch your conversasion earlier, please saperate the commit into these parts.

1. package.json info update.
2. add info in package.json.
3. add auto-update config

Thanks.

**Comment by pvnr0082t** [comment]
@x09326 I have separated commit into three parts. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t I can not recognize the difference from the first lines of the commits.

**Comment by pvnr0082t** [comment]
@x09326 I update the commit message. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
We should also add `*.fine-uploader/` as difference versions of this lib, btw @pvnr0082t you don't need to mention its author in every commits, it may be a little bit disturbing to the author.

**Comment by pvnr0082t** [comment]
@x09326 @PeterDaveHello There is no ```fineuploader.min.js``` after v3.7.0. Should the filename in package.json need to be changed to ```fine-uploader.min.js```?

**Comment by PeterDaveHello** [comment]
@pvnr0082t Since you didn't really update the new version, it's okay to leave it alone, or, you can add one more commit to add the latest version and update the filename at the same time.

**Comment by pvnr0082t** [comment]
@x09326 I have added the latest version and update the ```filename``` and ```version``` in package.json. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t the files you committed in v5.7.1 is inconsistent with the auto-update config.

**Comment by pvnr0082t** [comment]
> the files you committed in v5.7.1 is inconsistent with the auto-update config.

@PeterDaveHello Sorry, I don't get you. I run ```autoupdate.js``` to grab these files according to the 
auto-update config. Could you please explain more clear? Thanks.

**Comment by PeterDaveHello** [comment]
@pvnr0082t You committed s9me png images but not in autoupdate config.

**Comment by pvnr0082t** [comment]
@PeterDaveHello Actually, I'm not sure whether I should grab png images or not here. How can I confirm this?

**Comment by PeterDaveHello** [comment]
@pvnr0082t Maybe confirm with the author?

**Comment by pvnr0082t** [comment]
@x09326 I confirmed with author and the ```.png``` files need to be hosted. Here is the [reference](http://stackoverflow.com/questions/37178773/do-the-png-images-need-to-be-added-in-cdnjs). And I updated to the latest version __v5.8.0__. Please help me review again, thank you.

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
The commit message is too fat, they can be done in only one sentence, please try to simplify it, and, please don't cc the author when updating a lib, it's bothering, especially you usually need to rebase, update the commit, you won't like to receive that many notifications.

**Comment by pvnr0082t** [comment]
@x09326 I have simplified the commit message. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t nope, please 
```
update package.json information of Fineuploader …

update repository info, close #7707
```

->

```
Update repo info of Fineuploader, cc #7707
```

Please use single sentence for these commits, and do not close issue by the first commit, but the last.

**Comment by pvnr0082t** [comment]
@x09326 I have simplified the commit message and the latest version is v5.9.0 now. Please help me review again, thank you.

**Comment by rnicholus** [comment]
You may want to update before releasing this. Fine Uploader 5.9.0 was just released with a big license change. The project is now MIT.

**Comment by PeterDaveHello** [comment]
@pvnr0082t just add a correct / latest license of this project, we don't need to add an old one and update it, it's not like the old versions of lib.

**Comment by PeterDaveHello** [comment]
@rnicholus Thanks.

**Comment by pvnr0082t** [comment]
@x09326 I have added ```MIT``` license for the lib. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t you didn't add the png images correctly.

**Pull request merged**